packages/pluggableWidgets/rich-text-web/src/ui/RichText.scss

```diff
@@ -0,0 +1,40 @@
+@use "../../../node_modules/ckeditor4/contents.css";
+@use "../../../node_modules/ckeditor4/skins/moono-lisa/editor.css";
+@use "../../../node_modules/ckeditor4/plugins/tableselection/styles/tableselection.css";
+@use "../../../node_modules/ckeditor4/plugins/dialog/styles/dialog.css";
+@use "../../../node_modules/ckeditor4/plugins/copyformatting/styles/copyformatting.css";
+@use "./Scayt";
```

Comment:
Basically it's because of namespace. there are multiple `dialog.css` files in the import. But, what I did I just added different namespaces. Didn't know that scss has `as`